services/workflows-service/src/customer/customer.repository.ts

```diff
@@ -75,30 +62,30 @@ export class CustomerRepository {
     });
   }
 
-  async findByApiKey<T extends Omit<Prisma.CustomerFindFirstArgs, 'where'>>(
-    apiKey: string,
-  ): Promise<CustomerWithProjects | null> {
-    return this.prisma.customer.findFirst({
-      where: {
-        authenticationConfiguration: {
-          path: ['authValue'],
-          equals: apiKey,
-        },
-      },
-      select: {
-        id: true,
-        name: true,
-        authenticationConfiguration: true,
-        displayName: true,
-        logoImageUri: true,
-        faviconImageUri: true,
-        customerStatus: true,
-        country: true,
-        language: true,
-        projects: true,
-      },
-    });
-  }
+  // async findByApiKey<T extends Omit<Prisma.CustomerFindFirstArgs, 'where'>>(
```

Comment:
Why did you comment it out and not just deleted? We always have git's history if we'll need it...